README.md

```diff
@@ -34,9 +34,6 @@ change in NATS cluster name.
 
 \*** These versions are not compatible with Prometheus Alertmanager <= 0.24
 
-\**** This version is only compatible with Keptn 0.18.0 and potentially newer releases due to a breaking change with
-resource-service / configuration-service.
-
```

Comment:
I decided to remove this paragraph, as it's kind of implicit as there is a release for 0.16.x, 0.17.x and 0.18.x that should be used.